### ISSUE TITLE
Alien Cake Addict broken after PBR merge

### ISSUE THREAD

**Issue by rparrett** [bevyengine/bevy#1706]
## Bevy version

45b2db70705da24a89426e6b6e77d603a3983025

## Operating system & version

macOS

## What you did

`cargo run --example alien_cake_addict`

## What actually happened

```
wgpu error: Validation Error

Caused by:
    In Device::create_bind_group
    buffer binding size 64 is less than minimum 80

thread 'main' panicked at 'Handling wgpu errors as fatal by default', /Users/robparrett/.cargo/registry/src/github.com-1ecc6299db9ec823/wgpu-0.7.0/src/backend/direct.rs:1896:5
```

## Additional information

~~One~~Two other users in discord reported seeing this as well.

**Comment by alice-i-cecile** [comment]
Added to the milestone due to crashing regression.

**Comment by mtsr** [comment]
@JeanMertz spotted the error. I'm doing a quick run-through of the examples to check if they're all fixed and will PR then.

**Comment by CleanCut** [comment]
I ran into this today as well with the `contributors`, `sprite`, and `text2d` examples.  Some additional details in #1724.

**Comment by mtsr** [comment]
> I ran into this last night as well with the `contributors`, `sprite`, and `text2d` examples. Some additional details in #1724.

Those should also be fixed by #1708.

**Comment by CleanCut** [comment]
...and just like that, it's fixed. ❤️
